journal/updated/01-journal.adoc

```diff
@@ -1,146 +1,152 @@
+remained a secret pride and self-exaltation in most, if not all, of our formal performances.
+
+In this state and condition I knew not what to do. When neither writing,
+repeating sermons, reading of the scriptures and other good books,
 and sometimes expounding them to the best of my understanding, afforded me any comfort,
-I was at a loss, and knew not what way to take,
+I was at a loss, and knew not what way to take
 that I might have peace and comfort in my performances.
 Hitherto I knew not the Holy Spirit of the Lord, as I ought to have done,
 to be my leader and guide into all things that were necessary to my eternal salvation.
 
 Upon a certain time we had a meeting at Hugh David`'s, a tenant of Charles Lloyd`'s,
 of Dolobran, where one of our Independent teachers, who was a great scripturian,
-was preaching, and I wrote after him; and in his sermon he said,
+was preaching, while I wrote down his words after him. In his sermon he said,
 "`The time would come that there would be no need of the scriptures,
-any more than another book;`" at which I very much stumbled;
-and after the meeting I asked him, when would that time be?
-He said, when the Lord would make a new covenant with his people,
+any more than another book,`" at which I very much stumbled,
+and after the meeting I asked him when that time would be.
+He said, "`when the Lord would make a new covenant with His people,`"
 as it is said in Jeremiah 31:33-34,
 "`I will make a new covenant with the house of Israel; after those days, says the Lord,
-I will put my law in their inward parts, and write it in their hearts,
-and will be their God, and they shall be my people.
-And they shall teach no more every man his neighbour, and every man his brother, saying,
-"`Know the Lord; for they shall all know me,
+I will put My law in their inward parts, and write it in their hearts,
+and I will be their God, and they shall be My people.
+And they shall teach no more every man his neighbour,
+and every man his brother, saying, '`Know the Lord;`' for they shall all know Me,
 from the least of them unto the greatest of them, says the Lord;
 for I will forgive their iniquity,
-and I will remember their sin no more.`" It seems that he knew not that day come then,
-though he was a great preacher. I thought it would then be a happy day,
-when God would be the teacher of his people, himself;
-that we need not teach every man his neighbour, or his brother, saying,
-"`Know the Lord;`" but that we should all know him, from the least to the greatest.
-This day we knew not then, for all our preaching and long prayers;
-though the Lord did then beget true hungerings and thirstings in our souls after him.
-We had great love and zeal, and desired the knowledge of the truth, as it is in Jesus.
-Sometimes I have said, this was but like Jacob`'s dream, when he awoke and said,
+and I will remember their sin no more.`"
+It seems this man knew not then the day he spoke of,
```

Comment:
The meaning here is more like: It seems this man had not experienced that day to have already come, though he was a great preacher...